warehouse/accounts/services.py

```diff
@@ -75,6 +78,19 @@ def check_password(self, userid, password):
 
         return True
 
+    def create_user(self, user):
```

Comment:
I think I would like it better if this method took all of the data and actually created the new user itself, so it would have a call signature more like:

```python
def create_user(self, username, ...):
    user = User(username=username, ...)
    ...
    return user
```

I think this matches the other methods on this service better where it accepts parameters which are not specific to the fact we're storing users in a SQLAlchemy managed database but returns a newly created user object. We can probably get away without the ``find_userid`` check here too and let the calling code deal with checking if there is already a user by the name or not.